src/java/org/apache/cassandra/cql3/restrictions/Restriction.java

```diff
@@ -31,28 +31,32 @@
  */
 public interface Restriction
 {
-    public default boolean isOnToken()
+    /**
+     * Checks if this restriction is a token restriction.
+     * @return {@code true} if this restriction is a token restriction, {@code false} otherwise
+     */
+    default boolean isOnToken()
     {
         return false;
     }
 
     /**
-     * Returns the definition of the first column.
-     * @return the definition of the first column.
+     * Returns the column metadata in position order.
+     * @return the column metadata in position order.
```

Comment:
nit: Again, I guess this patch doesn't introduce it, but the description and `@returns` content seem redundant